docs/developer-experience/code-completion.md

```diff
@@ -0,0 +1,33 @@
+# Code Completion
+
+Code completion is a feature used in live programming that actively reads the user's current code in context to present code suggestions in real-time. The code suggestions may be intended to complete the code for efficiency or improve the code quality.
+
+## Code Completion Tools
+
+### 1. GitHub Co-Pilot: A Code Completion Tool
+
+![GitHub Code Pilot](./images/github-copilot.png)
+
+#### Benefits
+
```

Comment:
I thought you were going to add alternative code completion tools to the documentation etc.

Reply:
Add VSCode's Intellisense along with a downsides section for both.